### PR TITLE
Add generateOperationScopedModels flag

### PR DESCRIPTION
**Pull request by rashmihunt**
@raymondfeng PTAL

Related PRs
- https://github.com/strongloop/loopback-swagger/pull/99
- https://github.com/strongloop/loopback-component-explorer/pull/219

### CODE REVIEW

File: templates/projects/empty-server/data.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ template.server = {
       name: 'loopback-component-explorer',
       value: {
         mountPath: '/explorer',
+        generateOperationScopedModels: true,
```
Comment (raymondfeng) [inline templates/projects/empty-server/data.js]: Please add a comment about `generateOperationScopedModels`.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-workspace",
-  "version": "3.42.0",
+  "version": "4.0.0",
```
Comment (bajtos) [inline package.json]: We usually use a pre-release version number in a patch like this (e.g. `4.0.0-alpha.1`) and then let `slt-release` change it to the "production" version number.

Comment (rashmihunt) [reply package.json]: @bajtos done PTAL

File: templates/projects/empty-server/data.js (hunk 1 above)
Comment (rashmihunt) [reply templates/projects/empty-server/data.js]: @raymondfeng done PTAL

**Comment by rashmihunt** [comment]
@slnode test please

**Pull request merged**